dbfit-java/core/src/main/java/dbfit/util/sql/PreparedStatements.java

```diff
@@ -8,11 +8,18 @@ private static String buildParamList(int numberOfParameters) {
         return join(repeat("?", numberOfParameters), ", ");
     }
 
-    public static String buildStoredProcedureCall(String procName, int numberOfPararameters) {
-        return "{ call " + procName + "(" + buildParamList(numberOfPararameters) + ")}";
+    private static String buildReturnValue(boolean returnsValue) {
+        return returnsValue ? "? = " : "";
     }
 
-    public static String buildFunctionCall(String procName, int numberOfParameters) {
-        return "{ ? = call " + procName + "(" + buildParamList(numberOfParameters - 1) + ")}";
+    private static String buildRoutineInvocation(String routineName, int numberOfParameters) {
```

Comment:
Yes, I think I like that better. Or perhaps `executeRoutine`. I think any are fine. I'll push an update.